jep/223/README.adoc

```diff
@@ -269,6 +269,108 @@ The intent of this proposal is to improve overall security for Jenkins instances
+
+To do it we need to change the default value of getRequiredPermission() from:
+
+```
+public @CheckForNull Permission getRequiredPermission() {
+    return null;
+}
+```
+
+to default `Jenkins.ADMINISTER`:
+```
+public @CheckForNull Permission getRequiredPermission() {
+    return Jenkins.ADMINISTER;
+}
+```
+This change  only removes the links that are displayed in the "/manage" page for users that don't have the related permission.  Users with the `Jenkins.ADMINISTER` permission will not see any changes in the “/manage” page, and users with the `Jenkins.MANAGE` permission will only see links that have explicitly override the default implementation of `getRequiredPermission`.
+As this could break plugin compatibility, we did an analysis on the plugin usage of `getRequiredPermission` or `requiredPermission` in jelly/groovy.
+
+.Table Plugin usage analysis of ManagementLink.getRequiredPermission()
+|===
+|Plugin |Analyse |Reason
+
+|multi-slave-config-plugin
+|No change
+|getRequiredPermission() is overrided with `ADMINISTER` in the ManagementLink
+
+|scriptler-plugin
+|No change
+|Manages its own permission in the ManagementLink
+
+|p4-plugin
+|No change
+|Based on scriptler-plugin
+
+|periodic-jobs-balancer-plugin
+|No change
+|getRequiredPermission() is overrided with `ADMINISTER` in the ManagementLink
+
+|gerrit-trigger-plugin
+|No change
+|getRequiredPermission() is overrided with `ADMINISTER` in the ManagementLink
+
+|lenient-shutdown-plugin
+|No change
+|getRequiredPermission() is overrided with `ADMINISTER` in the ManagementLink
+
+|script-security-plugin
+|No change
+|getRequiredPermission() is overrided with `RUN_SCRIPT` in the ManagementLink
+
+|active-choices-plugin
+|No change
+|Based on scriptler-plugin
+
+|docker-plugin
+|Change
+|Does not override getRequiredPermission() in the ManagementLink and use it in jelly. The management page will be restricted to `ADMINISTER`.
+
+|libvirt-slave-plugin
+|Change
+|Does not override getRequiredPermission() in the ManagementLink and use it in jelly. The management page will be restricted to `ADMINISTER`.
+
+|build-with-parameters-plugin
+|No related
+|The requiredPermission is used from an Action class, not a ManagementLink class.
+
+|metadata-plugin
+|No related
+|The requiredPermission is used from an Action class, not a ManagementLink class.
+
+|flaky-test-handler-plugin
+|No related
+|The requiredPermission is used from an Action class, not a ManagementLink class.
+
+|ssh-slaves-plugin
+|No related
+|The requiredPermission is used from an Action class, not a ManagementLink class.
+
+|docker-traceability-plugin
+|No related
+|The requiredPermission is used from an Action class, not a ManagementLink class.
+
+|rebuild-plugin
+|No related
+|The requiredPermission is used from an Action class, not a ManagementLink class.
+
+|global-build-stats-plugin
+|No change
+|getRequiredPermission() is overrided with `ADMINISTER` in the ManagementLink
```

Comment:
The GlobalBuildStatsPlugin is a Plugin, it has a method get `requiredPermission`, but its internal ManagementLink has nothing there (and is not used in the view)